fs_drift_wrapper/ci_test.sh

```diff
@@ -23,5 +23,5 @@ cd ..
 
 indexes="ripsaw-fs-drift-results ripsaw-fs-drift-rsptimes ripsaw-fs-drift-rates-over-time"
 for i in $indexes ; do
```

Comment:
I don't think this for loop is needed given the check_es function loops over the indexes